relay-monitors/src/lib.rs

```diff
@@ -100,6 +100,17 @@ struct MonitorConfig {
     timezone: Option<String>,
 }
 
+// XXX(epurkhiser): This is a duplicate of the ClientSdkInfo that is part of the relay-general
+// crate. Until we're able to migrate the checkin payload over to it's own protocol using
+// metastructure we'll need to have this duplicated here
+//
+/// The SDK Interface describes the Sentry SDK and its configuration used to capture and transmit an event.
+#[derive(Debug, Deserialize, Serialize)]
+struct MinimalClientSdkInfo {
+    pub name: String,
+    pub version: String,
+}
```

Comment:
This is not strictly necessary, but if you want outdated external Relays to forward other attributes later on, it comes in handy:

```suggestion

    /// Other attributes for forward compatibility.
    #[serde(flatten)]
    other: BTreeMap<String, Value>,
}
```

Reply:
Does this mean they could shove as much as they want into the payload now however?

Reply:
For now this would be an overkill. Best to be added when moving to meta structure.